src/components/structures/RoomView.tsx

```diff
@@ -1578,7 +1578,6 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
         // XXX: why doesn't searching on name work?
 
         const ret = [];
```

Comment:
The changes in this file seem unnecessary, could they be please reverted to reduce the diff?

Reply:
Noted